model.lua

```diff
@@ -222,20 +222,20 @@ function Model:predict(dataloader, dtype)
     local ranks = torch.totable(ranks:double());
     for i = 1, #dataloader['unique_img_'..dtype] do
         -- rank list for all rounds in val split and last round in test split
-        if dtype == 'val' then
+        if dtype == 'test' then
```

Comment:
@satwikkottur This is similar to what you reviewed before, I hope this makes sense if someone needs to retrieve on train split itself.